.travis.yml

```diff
@@ -12,8 +12,11 @@ go_import_path: github.com/prometheus/prometheus
 # random issues on Travis.
 before_install:
 - travis_retry make deps
+- . $HOME/.nvm/nvm.sh
+- nvm install stable
+- nvm use stable
 - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then choco install make; fi
 
 script:
-- make check_license style unused test lint check_assets
+- make check_license style unused test react-app-test lint
```

Comment:
I have https://github.com/prometheus/prometheus/pull/5932 to "fix" the Makefile default target and have Circle CI run the `check_assets` target too. Any reason why the React tests can't run on Circle? Eventually I'd like us to keep only one CI system (eg Circle CI since it manages releases too).

Reply:
The React tests actually run fine on Circle, but the Prometheus orb on Circle just runs `make test`, and I didn't want to drag in the React tests into the normal `test` target yet to not unnecessarily annoy developers. Maybe I just should, since `make build` also requires node + yarn?

Reply:
Hmm, I don't think the Prometheus orb is involved here. I'm fine with adding an additional `make react-app-test` step to the Circle CI job if we don't want to extend `make test`.

https://github.com/prometheus/prometheus/blob/3c7028480271a70fac2d1958d6bb2fdf47b2b7de/.circleci/config.yml#L22-L38

Reply:
Ah right, it just calls `make` directly in the checked-in config. I did now just re-add the React tests to the normal `test` target since anyone building Prometheus will now need to have Node+Yarn installed anyway. I also removed the reference to `check_assets` that you added in a recent PR, since that target no longer exists.